Feat / new create and import command

Nullably opened the pull request:
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)


**Optional**:
- Related Github issue:
- Related Clubhouse Story: 8019


**A description of the changes proposed in the pull request**:
- New Create and Import commands to create/import configuration file.
- config command no longer works, config single variable command still works

mifeng commented:
`create` command should ask user what strategy and save the strategy in the config file. Currently, it assumes user is running PMM and doesn't ask the strategy question.

Nullably commented:
@mifeng I have added the strategy question to `create` command. Also removed strategy config variable from config files. The bot knows which strategy the config file is by reading the top 3 lines and scan for the strategy name.

mifeng commented on hummingbot/client/config/config_helpers.py:
```diff
@@ -191,21 +189,36 @@ def get_strategy_starter_file(strategy: str) -> Callable:
+            return strategy
+
+
+def strategy_matches_file(file_path, strategy) -> bool:
```
This method currently works, but in the future we plan to support users creating their own strategies or loading other custom strategies. Would defining a `strategy` param be better?

Nullably replied:
We simply need a kinda hard coded value in the file, it doesn't have to a config variable, just another entry in the template file will do. Though this will not be backward compatible as older version config files will not have this field. 
I'm fine with either approaches.

mifeng replied:
Let's require a `strategy` param in each config file that specifies the strategy name.

Since we're making so many changes to the client, the old config files won't be backward compatible with this new release anyway.

Nullably replied:
Yep, agree, probably will easier for other developers to understand the code base this way too.
I have made the changes to all strategy config files and their templates. Also fixed the problem with creating Arbitrage and XEMM strategy configs.

Nullably commented:
Run tests

Nullably commented:
run tests
